examples/example-kotlin/android/helloworld/app/src/main/res/layout/activity_helloworld.xml

```diff
@@ -1,21 +1,23 @@
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
-              xmlns:tools="http://schemas.android.com/tools" android:layout_width="match_parent"
-              android:layout_height="match_parent"
-              tools:context=".MainActivity"
-              android:orientation="vertical" >
+<LinearLayout
+        xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:tools="http://schemas.android.com/tools"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        tools:context=".MainActivity"
+        android:orientation="vertical" >
 
     <LinearLayout
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
             android:orientation="horizontal">
         <EditText
-                android:id="@+id/host_edit_text"
```

Comment:
We do not have a strictly defined guideline for resource ID naming, and believe the author's convention is "IDs suffixed with the name of the element in lowercase underscore". I found the most commonly used convention is to ["prefix with the name of the element in lowercase underscore"](https://stackoverflow.com/questions/35923922/best-practices-for-id-naming-conventions-in-android). This is an opinion-based answer. But, we would prefer name resource IDs in lowercase underscore, and we used the same convention in other Android examples in grpc-java repo.

Reply:
Done.
Thx for the review!